antora-playbook.yml

```diff
@@ -45,6 +45,9 @@ content:
     - url: git@github.com:apache/camel-spring-boot.git
       branches: master
       start_path: docs
+    - url: git@github.com:apache/camel-karaf.git
+      branches: master
+      start_path: docs
```

Comment:
This looks like an unrelated change, can you rebase to pick up the current playbook?